include/brahma.h

```diff
@@ -17,10 +19,6 @@ s32 firm_reboot ();
 #define ARM_RET     0xE12FFF1E // BX LR
 #define ARM_NOP     0xE1A00000 // NOP
 
-static u8  *g_ext_arm9_buf;
```

Comment:
Those variables are declared here, however are only used in brahma.c, so I moved them to the proper place.